lib/http.js

```diff
@@ -74,6 +74,36 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       res.json({meter});
     }));
 
+  // get meter config
+  app.get(
+    routes.meter,
+    cors(),
+    _addMeterRecordToRequest(),
+    _createAuthorizeMiddleware({
+      getExpectedTarget({req}) {
+        // only support update meter config URL as expected target; controller
+        // is the controller of the meter
+        const baseUrl = `https://${req.get('host')}${routes.basePath}`;
+        const expectedTarget =
+          `${baseUrl}/${encodeURIComponent(req.params.meterId)}`;
+        return {expectedTarget};
+      },
+      getRootController({req}) {
+        // root controller for update meter config is always the controller of
```

Comment:
```suggestion
        // root controller for get meter config is always the controller of
```

Reply:
The above helper functions seem like they could be moved below and reused in both get and update meter.